src/components/switch/index.tsx

```diff
@@ -56,6 +56,7 @@ export type SwitchProps = {
   thumbStyle?: StyleProp<ViewStyle>;
   style?: StyleProp<ViewStyle>;
   testID?: string;
+  id?: string;
```

Comment:
I guess it's a leftover, right?

Reply:
You can leave it here, please add it to the Switch's API file and I'll update the PR's changelog and name :)